app/src/main/java/de/westnordost/streetcomplete/settings/questselection/QuestSelectionFragment.kt

```diff
@@ -113,4 +128,16 @@ class QuestSelectionFragment
         questTypeOrderList.sort(questTypes)
         return questTypes.map { QuestVisibility(it, visibleQuestTypeController.isVisible(it)) }.toMutableList()
     }
+
+    private fun updateSubtitle() {
+        val subtitle = getSubtitle(requireContext(), questTypeRegistry, visibleQuestTypeController)
+        (requireActivity() as AppCompatActivity).supportActionBar?.subtitle = subtitle
+    }
+}
+
+fun getSubtitle(context: Context, questTypeRegistry: QuestTypeRegistry,
+                visibleQuestTypeController: VisibleQuestTypeController): String {
+    val enabledCount = questTypeRegistry.all.count { visibleQuestTypeController.isVisible(it) }
+    val totalCount = questTypeRegistry.all.size
+    return context.getString(R.string.pref_subtitle_quests, enabledCount, totalCount)
```

Comment:
I think it is best to inline that. This function only saves about 1 line of code to be shared and its name is so generic ("get subtitle of what??") that it may be confusing - after all you could import this global function anywhere.

If one wanted to reduce the logic here, I could come up with two things:
1. QuestTypeRegistry could implement the list interface itself instead of exposing `all` ... I think there was a neat syntax trick in Kotlin for that,
```kotlin 
QuestTypeRegistry(private val all: List<QuestType<*>>): List<QuestType<*>> by all
```  
or something like that

2. create extension function `QuestTypeRegistry.getAllVisible(visibleQuestTypeSource: VisibleQuestTypeSource): List<QuestType<*>>`